scripts/oqsprovider-pkcs12gen.sh

```diff
@@ -47,7 +51,7 @@ if [ $? -ne 0 ] || [ ! -f tmp/$1_srv_1.p12 ]; then
 fi
 
 # Generate config file with oqsprovider disabled
-sed -e 's/^oqsprovider/# oqsprovider/' "$(pwd)/scripts/openssl-ca.cnf" > tmp/openssl-ca-no-oqsprovider.cnf
+sed -e 's/^oqsprovider/# oqsprovider/' $OPENSSL_CONF > tmp/openssl-ca-no-oqsprovider.cnf
```

Comment:
I think sed will hang indefinitely if $OPENSSL_CONF is not defined, but before only a warning was printed. Perhaps $OPENSSL_CONF can fallback to "$(pwd)/scripts/openssl-ca.cnf" that is always available in the new if block?

Reply:
Thanks for this catch @iyanmv ! In the latest commit I opted for failing the test instead of setting a possibly incorrect fallback such as to discover setup problems early.

Reply:
Looks good!